README.md

```diff
@@ -203,31 +164,20 @@ Example:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
+👤 **Nilton Segura**
 
-👤 **Author1**
+-   GitHub: [@Nkaleth](https://github.com/Nkaleth)
+-   Twitter: [@NoeSeguraL](https://twitter.com/NoeSeguraL)
+-   LinkedIn: [LinkedIn](https://linkedin.com/in/nseguralu)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+-   Use ES6 syntax
```

Comment:
- [ ] Please make sure that you add the future features section to your readme and add at least 3 features that will be implemented in the future :smile: